verification/avocado_stub_util.py

```
@@ -0,0 +1,104 @@
+) -> dict[str, list[RenameData]]:
+    """Return the stub mappings for Avocado stubs for functions declared in the ANSI-C standard.
+
+    Args:
+        path_to_stub_mappings (str, optional): Path to stub mappings. Defaults to
+            DEFAULT_STUB_MAPPINGS.
+
+    Returns:
+        dict[str, RenameData]: A mapping from the original C function identifier to rename data.
+    """
+    with Path(path_to_stub_mappings).open(mode="rb") as f:
+        data = pkl.load(f)
```

Ruff (0.15.5)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)